### ISSUE TITLE
BUG: :title_row warns if the first row has fewer fields than the average row?

### ISSUE THREAD

**Issue by jpmckinney** [theodi/csvlint.rb#100]
That's the behavior. It seems like unusual behavior to me. Shouldn't it check if any row has more fields than the first row?

**Comment by nickzoic** [comment]
Yeah, I noticed that one as well ... it seems odd, but is there some reason behind it?
